test/functional/test_framework/test_framework.py

```diff
@@ -129,6 +130,8 @@ def main(self):
                             help="use bitcoin-cli instead of RPC for all commands")
         parser.add_argument("--perf", dest="perf", default=False, action="store_true",
                             help="profile running nodes with perf for the duration of the test")
+        parser.add_argument("--randseed", action="store", type=int,
```

Comment:
Perhaps just --seed for brevity

Reply:
I've gone the other way and changed this to `randomseed`. There's already a `portseed` argument, so more explicit is good (and I expect the `randomseed` argument to be used very infrequently so a few extra characters isn't an issue).